.circleci/config.yml

```diff
@@ -55,7 +55,17 @@ jobs:
       - run:
           name: Install RVM
           command: |
-            gpg --keyserver hkp://pool.sks-keyservers.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3 7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+            for key in 409B6B1796C275462A1703113804BB82D39DC0E3 \
+                       7D2BAF1CF37B13E2069D6956105BD0E739499BDB
+            do
+                for server in pgp.mit.edu keyserver.pgp.com ha.pool.sks-keyservers.net
```

Comment:
can you add a comment where you took these servers? for example some link like  https://rvm.io/rvm/security

Reply:
Sure. That's exactly where I took them. I've added a comment.